Frameworks/CoreText/CTFontDescriptor.mm

```diff
@@ -14,6 +14,9 @@
 //
 //******************************************************************************
 
+// TODO: File bug
```

Comment:
going to scrub my other 'todo file bug' comments too, before filing bugs.

---
In reply to: [95001484](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95001484) [](ancestors = 95001484,94999743)